src/teehr/fetching/s3/clone_from_s3.py

```diff
@@ -0,0 +1,157 @@
+from pathlib import Path
+import yaml
+import fsspec
+import pandas as pd
+import teehr.const as const
+from typing import Union, Literal
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def list_s3_evaluations(
+        format: Literal["pandas", "list"] = "pandas"
+) -> Union[list, pd.DataFrame]:
+    """List the evaluations in s3.
+
+    Parameters
+    ----------
+    format : str
+        The format of the output. Either "pandas" or "list".
+        Default is "pandas".
+
+    """
+    logger.info(f"Getting evaluations from s3: {const.S3_PROTOCOLS_PATH}")
+    # Read the content of the file using fsspec
+    with fsspec.open(const.S3_PROTOCOLS_PATH, 'r', anon=True) as file:
+        yaml_content = file.read()
+
+    # Load the YAML content into a dictionary
+    yaml_dict = yaml.safe_load(yaml_content)
+
+    if format == "pandas":
+        return pd.DataFrame(yaml_dict["protocols"])
+    return yaml_dict["protocols"]
+
+
+def clone_from_s3(ev, evaluation_name: str):
+    """Clone a study from s3.
+
+    Copies the study from s3 to the local directory.
+    Includes the following tables:
+        - units
+        - variables
+        - attributes
+        - configurations
+        - locations
+        - location_attributes
+        - location_crosswalks
+        - primary_timeseries
+        - secondary_timeseries
+        - joined_timeseries
+
+    Also includes the user_defined_fields.py script.
+
+    Parameters
+    ----------
+    ev : Evaluation
+        The Evaluation object.
+    evaluation_name : str
+        The name of the evaluation to clone.
+
+    Note: future version will allow subsetting the tables to clone.
+    """
+    logger.info(f"Cloning evaluation from s3: {evaluation_name}")
+    tables = [
+        {
+            "name": "units",
```

Comment:
It doesn't look like `name` is referenced in the rest of the code, is it a placeholder for future functionality?

Reply:
I think that was just for the humans.  Probably not really necessary.  Figured I'd use in a logger message er something but didn't.

Reply:
Added a logger line to use this.  Not sure it is necessary but feels like the dict should have a human readable key:value.